pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -355,6 +355,7 @@ func (c *Cluster) cephMgrOrchestratorModuleEnvs() []v1.EnvVar {
 		{Name: "ROOK_OPERATOR_NAMESPACE", Value: operatorNamespace},
 		{Name: "ROOK_CEPH_CLUSTER_CRD_VERSION", Value: cephv1.Version},
 		{Name: "ROOK_CEPH_CLUSTER_CRD_NAME", Value: c.clusterInfo.NamespacedName().Name},
+		{Name: "CEPH_ARGS", Value: fmt.Sprintf("--mon-host $(ROOK_CEPH_MON_HOST) --keyring %s", keyring.VolumeMount().KeyringFilePath())},
```

Comment:
DO we need `keyring` and `mon host` with same arg, can we have a separate env variable for it

Reply:
that are predefined but we can have them separate if radosgw-admin ceph cmd permits, https://github.com/rook/rook/blob/master/pkg/operator/ceph/csi/peermap/config.go#L218